views/templates/hook/linkblock.tpl

```diff
@@ -8,6 +8,7 @@
           class="{$link.class}"
           href="{$link.url}"
           title="{$link.description}"
+          {if isset($link.target) && $link.target} target="{$link.target}" {/if}
```

Comment:
I think you can use `!empty($link.target)` instead of `isset($link.target) && $link.target`